### PR TITLE
DOCSP-40957-1.43.3-release-notes

### PR DESCRIPTION
**Pull request by jason-price-mongodb**
## DESCRIPTION

Release notes per JIRA.

## STAGING

https://preview-mongodbjasonpricemongodb.gatsbyjs.io/compass/DOCSP-40957-1.43.3-release-notes/release-notes/#compass-1.43.3

## JIRA

https://jira.mongodb.org/browse/DOCSP-40957

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=667ee4edfe162e50060c8888

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks @jason-price-mongodb! LGTM

**Comment by jason-price-mongodb** [comment]
@addaleax please review. Thanks.

File: source/release-notes.txt [hunk 1]
```diff
@@ -10,6 +10,25 @@ Release Notes
...
+New Features:
+
+- Prevents modified tabs from being closed by accident
+  (:issue:`COMPASS-5022`).
+
+Bug Fixes:
+
+- Aggregations use ``maxTimeMS`` default on preview documents
+  (:issue:`COMPASS-7798`).
+- Ensures autoconnect information is provided in connection hooks
```
Comment (addaleax) [inline source/release-notes.txt:27]:
```suggestion
- Fix a regression that prevented autoconnect from working properly
```

**Review by addaleax — APPROVED** [review]
Just a suggestion for a small wording improvement ("connection hooks" are an implementation detail that users won't care about), LGTM either way

File: source/release-notes.txt (hunk 1 above)
Comment (jason-price-mongodb) [reply source/release-notes.txt:27]: Done.

**Pull request merged**